Simplenote/src/main/res/layout/activity_collaborators.xml

```diff
@@ -1,95 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/main_parent_view"
-    android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:layout_width="match_parent"
     android:orientation="vertical">
 
-    <include layout="@layout/toolbar" />
+    <include
+        layout="@layout/toolbar">
+    </include>
```

Comment:
1. Minor (🔍): Try running reformat on this layout file.
2. Minor (🔍): Try collapsing the tags when possible.

PS: Do that for every layout file created in this PR.

Reply:
Fixed in 9aa1abb289892f2998133c367f2242f060e00600.